client/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:18 as build-deps
+
+WORKDIR /opt/app
+
+COPY package*.json ./
+RUN npm clean-install
+
+COPY . .
+RUN npm run build
+
+FROM nginx as production
+EXPOSE 80
```

Comment:
The `nginx` base image we're extending _should_ have `EXPOSE` already defined for those:

```
$ docker inspect nginx | grep -i expose -A3
            "ExposedPorts": {
                "80/tcp": {}
            },
            "Tty": false,
--
            "ExposedPorts": {
                "80/tcp": {}
            },
            "Tty": false,
```



```suggestion
```